app/models/form/lettings/questions/managing_organisation.rb

```diff
@@ -45,9 +45,9 @@ def derived?
     true
   end
 
-  def hidden_in_check_answers?(_log, user = nil)
+  def hidden_in_check_answers?(log, user = nil)
     @current_user = user
-    @current_user.nil?
+    @current_user.nil? || !@page.routed_to?(log, user)
```

Comment:
Is there _ever_ a time that we want a value that can't be routed to in the CYA page?

Reply:
Hm, good question.
I think if we might have an opposite situation (If we have 2 questions on the page and one of them is conditional.
We still want to route to the page even if one of the questions doesn't show up)

But I can't think of any case where the situation you describe would happen. 
I've added the check to the default method too now.